### PR TITLE
Prepared React app, added MUI, added heading font support

### PR DESCRIPTION
**Pull request by Adamo-O**
## Brief Description
Prepared React app for development.

## Linked Issues
None

## Changes
- Added node dependencies
- Prepared React app for development
- Added CHANEY font
   - Used for headers h1, h2, h3
- Removed unnecessary files
- Commented out currently useless code (that may be of use in the future)
- Removed Meeting Minutes and Sprint 1 folders

### CODE REVIEW

**Review by KooshaG — DISMISSED** [review]
Looks good to me 🥴

File: online-shopping-website/package.json [hunk 1]
```diff
@@ -3,6 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.7.1",
```
Comment (Adamo-O) [inline online-shopping-website/package.json:6]: Added MUI dependencies

File: online-shopping-website/src/App.css [hunk 2]
```diff
@@ -1,38 +1,3 @@
...
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
-}
+.App-content {
+  padding: 0.5em;
+}
```
Comment (Adamo-O) [inline online-shopping-website/src/App.css:3]: Removed default React styles

File: online-shopping-website/src/App.js [hunk 3]
```diff
@@ -1,23 +1,11 @@
...
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="App-content">
```
Comment (Adamo-O) [inline online-shopping-website/src/App.js:5]: Removed default React styles, added test heading

File: online-shopping-website/src/index.css [hunk 4]
```diff
@@ -1,13 +1,18 @@
+/* Specifying font families and typography */
+@font-face {
+  font-family: 'CHANEY';
+  src: local('CHANEY'), url(./fonts/CHANEY-Regular.otf) format('opentype');
+}
+
+h1, h2, h3 {
+  font-family: 'CHANEY';
+}
+
```
Comment (Adamo-O) [inline online-shopping-website/src/index.css:10]: Added support for CHANEY font, and made it the default heading font

File: online-shopping-website/src/index.js [hunk 5]
```diff
@@ -2,16 +2,19 @@ import React from 'react';
...
-import reportWebVitals from './reportWebVitals';
+import { StyledEngineProvider } from '@mui/material/styles';
+// import reportWebVitals from './reportWebVitals';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <StyledEngineProvider injectFirst>
+      <App />
+    </StyledEngineProvider>
```
Comment (Adamo-O) [inline online-shopping-website/src/index.js:12]: Using `StyledEngineProvider` to overwrite MUI styles

**Review by aspyridakos — APPROVED** [review]
Looks good! 🚀

**Review by KooshaG — APPROVED** [review]

**Pull request merged**